.github/workflows/tag.yml

```
@@ -0,0 +1,104 @@
+# NB: release notes inspired by https://blogs.sap.com/2018/06/22/generating-release-notes-from-git-commit-messages-using-basic-shell-commands-gitgrep/
+name: Tag Creator
+on:
+  issues:
+    types: [opened]
+jobs:
+  bumpversion:
+    if: >-
+      (
+        startsWith( github.event.issue.title, '[bumpversion:major]' ) ||
+        startsWith( github.event.issue.title, '[bumpversion:minor]' ) ||
+        startsWith( github.event.issue.title, '[bumpversion:patch]' )
+      ) && (
+        github.event.issue.user.login == 'lukasheinrich' ||
+        github.event.issue.user.login == 'matthewfeickert' ||
+        github.event.issue.user.login == 'kratsg'
+      )
+    env:
+      IS_MAJOR: >-
+          ${{ startsWith( github.event.issue.title, '[bumpversion:major]' ) }}
+      IS_MINOR: >-
+          ${{ startsWith( github.event.issue.title, '[bumpversion:minor]' ) }}
+      IS_PATCH: >-
+          ${{ startsWith( github.event.issue.title, '[bumpversion:patch]' ) }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: determine part
```

```suggestion
    - name: Determine part
```

Some of the step names are captialized, but not all, so go all start with caps?